bin/main.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+'''Copyright (C) 2017 Collabora
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Authors: Shane Fagan - shane.fagan@collabora.com
+'''
+
+import datetime
+import sys
+import os
+from time import sleep
+from threading import Thread
+sys.path.append(os.path.dirname(__file__))
+RULES_PATH = os.path.abspath(os.path.join('..', 'rules', 'simple'))
```

Comment:
Well I was saving it for later, I was planning on putting the config file in the rules dir and grabbing it. I'll handle it with parms instead if that is the approach you want.